### PR TITLE
fix: correctly parse arguments for list --is-pack

### LINKED ISSUE #186: `asar list` may lose some output if it's writing to a pipe
`asar list example.asar | less` may not produce the complete output under Linux.

I believe `Process.exit(0)` should not be called after `console.log`.

See https://nodejs.org/api/process.html#process_process_exit_code and https://nodejs.org/api/process.html#process_a_note_on_process_i_o

### PR DESCRIPTION
**Pull request by malept**
build(deps): upgrade commander to ^5.0.0

Fixes #186.

### CODE REVIEW

File: test/cli-spec.js [hunk 1]
```diff
@@ -118,7 +118,7 @@ describe('command line interface', function () {
     return assertAsarOutputMatches('l test/expected/packthis-unpack-dir.asar', 'test/expected/extractthis-filelist.txt')
   })
   it('should list files/dirs in archive with unpacked dirs & is-pack option', async () => {
-    return assertAsarOutputMatches('l test/expected/packthis-unpack-dir.asar --is-pack', 'test/expected/extractthis-filelist-with-option.txt')
+    return assertAsarOutputMatches('l --is-pack test/expected/packthis-unpack-dir.asar', 'test/expected/extractthis-filelist-with-option.txt')
```
Comment (malept) [inline test/cli-spec.js]: Swapping the position of the flag shows the bug with commander 4.x.

File: package.json [hunk 2]
```diff
@@ -44,7 +44,7 @@
   },
   "dependencies": {
     "chromium-pickle-js": "^0.2.0",
-    "commander": "^4.1.1",
+    "commander": "^5.0.0",
     "glob": "^7.1.6",
     "minimatch": "^3.0.4"
```
Comment (Philhoy54) [inline package.json:49]: const { app, globalShortcut } = require('electron')

app.whenReady().then(() => {
  // Register a 'CommandOrControl+Y' shortcut listener.
  globalShortcut.register('CommandOrControl+Y', () => {
    // Do stuff when Y and either Command/Control is pressed.
  })
})

Comment (Philhoy54) [reply package.json:49]: Super

File: test/cli-spec.js [hunk 3]
```diff
@@ -118,7 +118,7 @@ describe('command line interface', function () {
...
   })
   it('should extract an archive with unpacked dirs', async () => {
     await execAsar('e test/input/extractthis-unpack-dir.asar tmp/extractthis-unpack-dir/')
```
Comment (Philhoy54) [inline test/cli-spec.js:124]: @electron-bot @malept

**Comment by Philhoy54** [comment]
image

**Pull request merged**